src/redrose-launcher/main.sh

```
@@ -0,0 +1,80 @@
+    ((index=index+1))
+    trim=$(echo "$line" | tr -d '"\\')
+    #versions
+    echo "$index - $trim"
+done <<< "$versions"
+
+read -p "Which version to download? (1) -> " answer
+
+
+if [ "$answer" = "" ]; then
+    url=$(echo "$res" | jq -r ".[0].assets.[].browser_download_url")
+    download_url $url
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)